NorimaConsulting/training-and-onboarding
# REC-46/31  Single Recipe Page

### What it is/Resolves: A recipe page to display a selected recipe that the user has selected, with all the needed information to make it. 

### Testing: 

1. cd into webapp and run pnpm dev and go to http://localhost:3000
2. Click on any of the 3 recipes displayed on the Homepage. Once clicked, a new tab should open with the single recipe. 
3. Recipe should have: Posted by, Title, Image, Instructions, Ingredients and Rating. 
4. Go to the Recipe List: NavBar > Recipes 
5. Click on any of the recipes displayed in that list. Once clicked, a new tab should open with the single recipe. 

#### What it should look like: 
![image](https://user-images.githubusercontent.com/83307690/150463454-dcdc5329-0f7a-41e8-9bec-d5da0c24a829.png)

![image](https://user-images.githubusercontent.com/83307690/150463481-5ea750be-8f83-4f6f-ad85-387da2222632.png)
